packages/frontend-api/src/Model/Resolver/Products/ProductByUuidResolver.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Resolver\Products;
+
+use Overblog\GraphQLBundle\Definition\Resolver\AliasedInterface;
+use Overblog\GraphQLBundle\Definition\Resolver\ResolverInterface;
+use Overblog\GraphQLBundle\Error\UserError;
+use Ramsey\Uuid\Uuid;
+use Shopsys\FrameworkBundle\Model\Product\Exception\ProductNotFoundException;
+use Shopsys\FrameworkBundle\Model\Product\ProductElasticsearchProvider;
+
+class ProductByUuidResolver implements ResolverInterface, AliasedInterface
+{
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Product\ProductElasticsearchProvider
+     */
+    protected ProductElasticsearchProvider $productElasticsearchProvider;
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Product\ProductElasticsearchProvider $productElasticsearchProvider
+     */
+    public function __construct(
+        ProductElasticsearchProvider $productElasticsearchProvider
+    ) {
+        $this->productElasticsearchProvider = $productElasticsearchProvider;
+    }
+
+    /**
+     * @param string $uuid
+     * @return array
+     */
+    public function resolver(string $uuid): array
+    {
+        if (Uuid::isValid($uuid) === false) {
```

Comment:
This check should not be neccessary in case you have it defined in `*.type.yaml`

Reply:
oh, nice catch! fixed in c16720fe9